install_nim.sh

```diff
@@ -74,13 +60,6 @@ if [[ "$nim_version" = "devel" ]]; then
   exit
 fi
 
-# fetch latest tag if 'nim_version' was 'stable'
-if [[ "$nim_version" = "stable" ]]; then
-  # NOTE: jq is pre-installed on github actions runner
-  nim_version="$(fetch_tags | latest_version)"
-elif [[ "$nim_version" =~ ^[0-9]+\.[0-9]+\.x$ ]] || [[ "$nim_version" =~ ^[0-9]+\.x$ ]]; then
```

Comment:
Please do not delete this block. This block need to process patterns like `1.6.x` and `2.x`. 

```bash
if [[ "$nim_version" =~ ^[0-9]+\.[0-9]+\.x$ ]] || [[ "$nim_version" =~ ^[0-9]+\.x$ ]]; then
  nim_version="$(fetch_tags | grep -E "$(tag_regexp "$nim_version")" | latest_version)"
fi
```

Reply:
You are right! I was wondering why the CI was failing. Sorry for the confusion.